Cairo Desktop/CairoDesktop.Configuration/app.config

```diff
@@ -109,6 +109,21 @@
             <setting name="TaskbarMiddleClick" serializeAs="String">
                 <value>0</value>
             </setting>
+            <setting name="DesktopBackgroundType" serializeAs="String">
+                <value>windowsDefaultBackground</value>
```

Comment:
The type/style settings seem more like enum type values, would it make sense to store them as an integer like some of the other settings?

Reply:
BackgroundType i wanted to be a string so that we can extend this in the future. using an enum would create a limitation in this case.  Wallpaperstyle can be an int.